gulpjs/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Use the base App Engine Docker image, based on debian jessie.
+FROM debian:8
+
+# Install updates and dependencies
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick && \
+    apt-get clean && rm /var/lib/apt/lists/*_*
+
+# Install the latest LTS release of nodejs
+RUN mkdir /nodejs && curl https://nodejs.org/dist/v6.9.1/node-v6.9.1-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
+ENV PATH $PATH:/nodejs/bin
+RUN npm install -g gulp-cli
```

Comment:
@jmdobry @ofrobots 

Typically people will install gulp as a dev dependency in their package.json.  If you install dev dependencies, it also symlinks the entrypoint so you can just invoke `gulp` in an npm script.  This is important for producing hermetic builds, and also so you can pin to a specific version of gulp that's not head.  We should never `npm install -g` a dev dependency like this for a user.  

On other words - I don't think this is the right way to make this builder at all.  A proper builder for node.js can be built with a single multi-purpose pipeline. 

Proposed Steps
- Run `npm install` (notice that I didn't add --production).  
- Run some user defined build step, possibly by convention:  `npm run build`
- Build the container
- Entrypoint is set to `npm start`

If we're feeling super fancy we could do a two container process where we trim the npm dependencies down to only the --prod ones in the actual runtime container. 

@ofrobots - this really intersects with the builder work y'all are already doing.  We should sync.